nebula_decoders/src/nebula_decoders_velodyne/decoders/vls128_decoder.cpp

```diff
@@ -252,7 +252,6 @@ void Vls128Decoder::unpack(const velodyne_msgs::msg::VelodynePacket & velodyne_p
           float azimuth_corrected_f = azimuth + (azimuth_diff * vls_128_laser_azimuth_cache_[firing_order]) - corrections.rot_correction * 180.0 / M_PI * 100;
           
           if (azimuth_corrected_f < 0.0){
-            // std::cout << "azimuth_corrected_f: " << azimuth_corrected_f << std::endl;
             azimuth_corrected_f += 36000.0;
           }
           // const int16_t azimuth_corrected = ((int16_t)round(azimuth_corrected_f)) % 36000;
```

Comment:
This one as well please!

Reply:
I forgot this. I fixed it right now. Thank you!